### PR TITLE
Merge 1.1.x into 1.x

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by dwijnand** [comment]
Blocked on a CI problem: https://github.com/sbt/zinc/issues/479

**Review by eed3si9n — APPROVED** [review]
LGTM pending CI

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?zinc=pull/478/head&deploy_to_bintray=false&run_sbt_community=false">this link.</A>

**Pull request merged**
